.docker/hudi/data/load.py

```diff
@@ -0,0 +1,96 @@
+from dataclasses import dataclass
+from pathlib import Path
+from pyspark.sql import SparkSession
+from pyspark.sql.types import (
+    StructType,
+    StructField,
+    IntegerType,
+    DecimalType,
+    TimestampType,
+)
+
+
+@dataclass
+class DataSource:
+    filename: str
+    hudi_options: dict
+    schema: StructType
+
+
+SOURCES = [
+    DataSource(
+        filename="payments.csv",
+        hudi_options = {
+            "hoodie.table.name": "payment",
+            "hoodie.datasource.write.recordkey.field": "payment_id",
+        },
+        schema=StructType([
+            StructField("payment_id", IntegerType(), nullable=False),
+            StructField("customer_id", IntegerType(), nullable=False),
+            StructField("staff_id", IntegerType(), nullable=False),
+            StructField("rental_id", IntegerType(), nullable=False),
+            StructField("amount", DecimalType(), nullable=False),
+            StructField("payment_date", TimestampType(), nullable=False),
+        ]),
+    ),
+]
```

Comment:
Adapting this script just requires changing the `DataSource`s in this list. The rest of the script is table/file-agnostic.